core/src/test/java/org/elasticsearch/index/mapper/TextFieldTypeTests.java

```diff
@@ -82,18 +82,19 @@ public void testTermQuery() {
         assertEquals("Cannot search on field [field] since it is not indexed.", e.getMessage());
     }
 
-    public void testTermsQuery() {
+    //We still need this test??
```

Comment:
yes, please. Lucene renamed the `TermsQuery` to `TermInSetQuery` but we plan on keeping support for the `terms` query in Elasticsearch, which is what this test is testing.